### PR TITLE
Alternative schema of AmountAdjustment

### PR DESCRIPTION
**Pull request by slavcodev**

### CODE REVIEW

**Comment by vova07** [comment]
Please review, it is ready to merge now.

File: spec/definitions/Dunning/Policies/discount.yaml [hunk 1]
```diff
@@ -2,8 +2,12 @@ allOf:
   - $ref: "#/definitions/AmountAdjustment"
   - type: object
     properties:
-      rate:
-        description: The rate of payment amount
-        type: string
+      value:
+        description: The payment amount discount
+        type: integer
```
Comment (slavcodev) [inline spec/definitions/Dunning/Policies/discount.yaml]:
~~~
type: number
format: float
~~~

?

**Comment by vova07** [comment]
Please review, it is ready to merge now.

**Comment by XaosSintez** [comment]
It looks ok for me

**Comment by vova07** [comment]
Hope some of native Enghilsh speakers will can propose a better name for `fixed` attribute.
The idea of this attribute is to tell the merchant that if the value is `true` the system will consider it as `float value` if `false` it will be used as `percent value`. 
Unfortunately my english it's not good enough to propose something better than `fixed`.

**Pull request merged**